.github/CONTRIBUTING.md

```diff
@@ -188,22 +207,52 @@ lines included in `<remarks>`.
 
 Public serialized parameters that appear in the inspector also require
 XML comments and an additional `[Tooltip("")]` which is displayed in
-the Unity Editor inspector panel. The order of documentation and
-parameter attributes should be:
+the Unity Editor inspector panel. All parameter attributes should
+appear in the same attribute block and comma separated with the
+`Tooltip` attribute first followed by the remaining attributes in an
+order that makes most sense when read as a sentence. The order of
+documentation and parameter attributes should be:
 
   * XML documentation
-  * Optional attributes
-  * Tooltip attribute
+  * Parameter attributes
 
 ```
 /// <summary>
 /// Description here.
 /// </summary>
-[Optional-Attributes...]
-[Tooltip("Description here.")]
+[Tooltip("Description here."), InternalSetting, SerializeField]

```

Comment:
There is no need for `SerializeField` on `public` fields and therefore it should not be listed.

Either change the example here to `protected` or drop the `SerializeField` annotation.

Reply:
I'll change the example to protected